### PR TITLE
Login & Registration

### PR DESCRIPTION
**Pull request by olgapolinkina**
### Checklist
- [x] I've run `mvn test` from the root directory to see all new and existing tests pass
- [x] I've followed code style and run and ensured the code style is valid
- [x] I've created new tests if necessary.

### Motivation and Context
Login & Registration part finished and ready to be moved to production part.
Issue list: #2, #3, #7, #8, #25 
Tested:
   * in jUnit using mockito: checks that passed data corresponds to userStore.getCurrentUser() response.
   * in Browser: all links do what they suppose.

### Description
1. Changed index webpage (changed background image, and added counter of days left till "07-09-2018"
2. Added new fields to User Entity & weppage -> email & phone
3. Front-End adds new fields to preexisting template and sends them to Back-end.
4. Back end create new User with UserDto from request data.
5. Mockita's test checks that userStore.getCurrentUser() return User data with additional two fields.

### CODE REVIEW

File: src/main/java/lv/ctco/javaschool/auth/entity/domain/User.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package lv.ctco.javaschool.auth.entity.domain;
 
 import javax.persistence.*;
```
Comment (St-Filer) [inline src/main/java/lv/ctco/javaschool/auth/entity/domain/User.java]: Please avoid asterisk imports

File: src/test/java/lv/ctco/javaschool/auth/control/UserStoreTest.java [hunk 2]
```diff
@@ -10,6 +13,48 @@
 
 class UserStoreTest {
 
+   /*
```
Comment (St-Filer) [inline src/test/java/lv/ctco/javaschool/auth/control/UserStoreTest.java]: Please remove commented code.

File: src/main/java/lv/ctco/javaschool/auth/boundary/AuthenticationApi.java [hunk 3]
```diff
@@ -59,10 +59,12 @@ public Response login(UserLoginDto userLogin, @Context HttpServletRequest reques
     public Response register(UserLoginDto userLogin, @Context HttpServletRequest request, @Context HttpServletResponse response) {
```
Comment (St-Filer) [inline src/main/java/lv/ctco/javaschool/auth/boundary/AuthenticationApi.java]: Tests for `AuthenticationApi`?

Comment (olgapolinkina) [reply src/main/java/lv/ctco/javaschool/auth/boundary/AuthenticationApi.java]: AuthenticationApiTest - added.
Cchecks only method added to Login&Register template

**Pull request merged**